src/main/java/pl/touk/parking/repository/ParkingUsageRepository.java

```diff
@@ -0,0 +1,17 @@
+package pl.touk.parking.repository;
+
+import org.springframework.data.repository.Repository;
+import pl.touk.parking.domain.ParkingUsage;
+
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Optional;
+
+public interface ParkingUsageRepository extends Repository<ParkingUsage, Long> {
+
+    void save(ParkingUsage usage);
+
+    Optional<ParkingUsage> findByDriverVehicleNumberAndEndTimeIsNull(String vehicleNumber);
+
+    Collection<ParkingUsage> findAllByStartTimeGreaterThanEqualAndEndTimeBefore(LocalDateTime start, LocalDateTime end);
```

Comment:
rozumiem Spring Data JPA i dlatego musiałeś napisać takie nazwy. Ale czy dałoby się to zrobić czytelniej?